astropy/tests/disable_internet.py

```diff
@@ -52,7 +52,7 @@ def new_function(*args, **kwargs):
             valid_hosts = ('localhost', '127.0.0.1')
 
         if allow_astropy_data:
-            for valid_host in ('data.astropy.org', 'astropy.stsci.edu'):
+            for valid_host in ('data.astropy.org', 'astropy.stsci.edu', 'www.astropy.org'):
```

Comment:
Maybe we want to consider to add the github astropy-data repo as well as the last resort to access the astropy data?

Reply:
Just to clarify, this is not the list of mirrors - the mirrors are defined here:

https://github.com/astropy/astropy/blob/master/astropy/utils/data.py#L47

and the second one is the astropy-data repo.